gem/Gemfile

```diff
@@ -16,8 +16,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
-gem 'coveralls', require: false
-gem 'rspec'
-gem 'rubocop'
-gem 'thor' # enable: bundle exec thor spec
```

Comment:
update: as the gem isn't actually using thor for task running / coveralls for code coverage then it's fine to keep these deleted.